lightstep/resource_metric_condition.go

```diff
@@ -344,6 +312,66 @@ func getThresholdSchema() map[string]*schema.Schema {
 	}
 }
 
+// expression is optional because it cannot be included in composite alerts but is otherwise required
+func getUnifiedAlertExpressionSchema() *schema.Schema {
+	resource := getCompositeSubAlertExpressionResource()
+	resource.Schema["is_multi"] = getIsMultiSchema()
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem:     resource,
+	}
+}
+
+// expression is required in legacy metric conditions
+func getMetricConditionExpressionSchema() *schema.Schema {
+	resource := getCompositeSubAlertExpressionResource()
+	resource.Schema["is_multi"] = getIsMultiSchema()
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Required: true,
+		MaxItems: 1,
+		MinItems: 1,
+		Elem:     resource,
+	}
+}
```

Comment:
these are the separate paths of the branch above and they're quite similar, the difference is Required/Optional & MinItems. I think breaking things out this way will make a bit more sense when I add the `SubAlert` version